src/elements/content-sidebar/activity-feed/activity-feed/ActivityItem.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../../common/variables';
+
+.bcs-ActivityItem {
+    .bcs-Comment {
```

Comment:
I'm not too familiar with Sidebar, is the `Comment` component used in other places? is this nesting necessary? why not move it to `Comment.scss`?

Reply:
Because this padding is specific to `Activity`, we want decouple as much activity related css away from the comment component if possible